app/models/course/assessment/answer.rb

```diff
@@ -24,6 +24,8 @@ class Course::Assessment::Answer < ActiveRecord::Base
     end
     state :graded do
       event :unsubmit, transitions_to: :attempting
+      # Does nothing but revert the state, for the case we want to keep the grading info
+      event :unmark, transitions_to: :submitted
```

Comment:
Actually the graded answer could come from either state ( submitted or evaluated ), but I guess evaluated is more common, will do the change.